CSI_Greenville_HIST8510_Final_Project-AR.qmd

```diff
@@ -0,0 +1,390 @@
+
+```{r}
+#load CSI:Dixie Greenville County data
+CSI.GVL <- read.csv("https://docs.google.com/spreadsheets/d/e/2PACX-1vQzO3uN1w0-A_rN3bHuBWDr1iQe9hPZ68u7igu-I8wWDEpHSWH5T3AQFPvaa-VMSbuB8Z8X5wxSjmA6/pub?output=csv") 
+```
+
+```{r}
+#tidy data
+CSI.GVL.TIDY <- CSI.GVL %>%
+  separate(NAME, c("FIRST.NAME", "LAST.NAME"), sep = " ", remove = FALSE) %>% #separates first and last names
+  separate(DATE, c("MONTH", "DAY", "YEAR"), sep = " ", remove = FALSE) #separates dates into month, day, year
+```
+
+**CSI:Greenville**
+
+Greenville County was officially created in 1786. The land ceded from the Cherokee Nation after the Revolutionary War as well as lands confiscated from Loyalists became available for sale through the land grant office and was quickly bought up by newly independent Americans eager to reproduce the agricultural wealth of the lowcountry of South Carolina. The 1820s brought tremendous change as infrastructure became the focus of Greenville and so much of the state and nation as a whole. The State Road project was funded, connecting Greenville to Columbia and Charleston, as well as many bridge, ferry crossing and canal projects.
+
+As South Carolina moved toward the middle of the 19th century, political strife began to characterize much of its residents' interactions. Duels between white men were very often threatened, and (less often) occurred along the riverbanks and in old fields throughout the county. Despite the political turmoil of Nullification and Secession, industry continued to develop. The fast moving rivers coming out of the mountains were well suited to mill creation and the wealthy men able to purchase large tracts of land prized areas that lent themselves to diversified economic pursuits.
+
+Until the beginning of the Civil War Greenville's most important commodity was its location at the base of the Appalachians. The drover trade coming from Tennessee and Kentucky with livestock and other goods had a sophisticated network for alerting the livestock yards and merchants selling food and rest of their impending arrival. From the coast, the elite Charlestonians and lowcounty residents flocked to Greenville every summer to escape the disease and heat of the coast.
+
+With the federal census reforms in 1850 there is an immediate impact on the data available in Greenville County. Coroner inquests increased dramatically from 1850 through the end of the 1880s, with the 1870s and 80s having the highest occurrences by a significant margin.
+
+```{r}
+#chart Greenville by year
+year.count <- CSI.GVL.TIDY %>%
+  group_by(YEAR) %>%
+  summarize(count = n())
+  
+ggplot(year.count, aes(x = YEAR, y = count)) +  geom_point(aes(size = count)) + theme(axis.text.x=element_text(angle=90,hjust=1)) + labs (title = "Greenville Inquests by Year")
```

Comment:
This isn't necessarily the right chart for what you are trying to demonstrate. You have count represented both on the y-axis but also in the size of the dot. A line chart would be more effective